scipy/spatial/kdtree.py

```diff
@@ -199,6 +199,10 @@ class KDTree(object):
             >>> import sys
             >>> sys.setrecursionlimit(10000)
 
+    See Also
+    --------
+    cKDTree : Implementation of `KDTree` in Cython
```

Comment:
The implementation is here:
https://github.com/scipy/scipy/tree/master/scipy/spatial/ckdtree/src